python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -464,3 +466,66 @@ def hybrid_forward(self, F, x):
 
     def __repr__(self):
         return self.__class__.__name__
+
+
+class Lambda(HybridBlock):
+    r"""Wraps an operator or an expression as a HybridBlock object.
+
+
+    Parameters
+    ----------
+    function : str or function
+        Function used in lambda must be one of the following:
+        1) the name of an operator that is available in both symbol and ndarray
+        2) an operator in either symbol or ndarray, and is shared by symbol and ndarray.
+        3) a function that conforms to "def function(data, *args, **kwargs)".
```

Comment:
lambda should have def function(*args)
hybrid lambda should have def function(F, *args)
no kwargs

Reply:
Just to confirm, is the intention to keep consistent with HybridBlocl? kwargs should work here unlike HybridBlock, since it doesn’t go through __call__

Reply:
def function(F, *args) is easier to understand